### DISCUSSION TITLE
How to mosaic the result inversed by mintpy?

### DISCUSSION THREAD

**Discussion by gaiyf** [insarlab/MintPy#918] [Q&A] [answered]
I think there is a command but i cannot find the tutorial page.

**Comment by Carolinah23** [comment, marked as answer]
I think you can use the tool `image_stitch.py` incorporated in mintpy. 

I haven't used it myself but from the help section, I understand the results should have a common area. However, I don't know how it deals with the reference points in each dataset.

**Comment by mosyhey** [comment]
[Here](https://github.com/insarlab/MintPy-tutorial/) is the tutorial. Mosaiking multiple geocoded files, is briefly explained at the end of [this page](https://github.com/insarlab/MintPy-tutorial/blob/main/smallbaselineApp_aria.ipynb).
